packages/module/patternfly-docs/content/extensions/chatbot/examples/UI/UI.md

```diff
@@ -335,10 +337,22 @@ Actions can be added to conversations with `menuItems`. Optionally, you can also
 
 ```
 
+## Modals
+
 ### Modal
 
 Based on the [PatternFly modal](/components/modal), this modal adapts to the chatbot display mode and accepts components typically used in a modal. It is primarily used and tested in the context of the attachment modals, but you can customize this modal to adapt it to other use cases as needed. The modal will overlay the chatbot in default and docked modes, and will behave more like a traditional PatternFly modal in fullscreen and embedded modes.
 
 ```js file="./ChatbotModal.tsx" isFullscreen
 
 ```
+
+### Terms of Use
+
+Based on the [PatternFly modal](/components/modal), this modal adapts to the ChatBot display mode and is meant to display terms and conditions for using a ChatBot in your project. The image in the header can be toggled on or off depending on whether the `image` and `altText` props are provided.
+
+This example also includes an example of how to use [skip to content](/patternfly-ai/chatbot/ui#skip-to-content). When the terms of use modal is open, focus is placed on the terms of use container. When it is closed, focus is placed on the chatbot. In a real example with a functioning toggle, you would also want to place focus on the toggle when appropriate.
```

Comment:
```suggestion
### Terms of use

Based on the [PatternFly modal](/components/modal), this modal adapts to the ChatBot display mode and is meant to display terms and conditions for using a ChatBot in your project. The image in the header can be toggled on or off depending on whether the `image` and `altText` props are provided.

This example also includes an example of how to use [skip to content](/patternfly-ai/chatbot/ui#skip-to-content). When the terms of use modal is open, focus is placed on the terms of use container. When it is closed, focus is placed on the ChatBot. In a real example with a functioning ChatBot toggle, you would also want to place focus on the toggle when appropriate.
```

is this the only example that displays the focus behavior you outlined? wondering if these docs would feel better in a more general example, since users would probably only come to this one when using the terms modal. But I'm not sure if the behavior is fully covered elsewhere since this is the example that expands on modal use this much. If this is the only place, then that's okay

Reply:
This is the only example because it's part of a user flow they'd have to go through before using the chatbot. (They'd have to potentially accept the terms first, so I want to make sure that you can skip to it too and document how to do it.)

Reply:
that makes sense!